.github/workflows/automate_merge_renovate_prs.yaml

```
@@ -0,0 +1,85 @@
+name: Automate Merge Renovate PRs
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '*/10 * * * *'
+
+jobs:
+  diff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const owner = "backstage";
+            const repo = "backstage";
+            const query = `{
+              repository(name: "backstage", owner: "backstage") {
```

```suggestion
              repository(owner: "backstage", name: "backstage") {
```

I claim the useless nit of the year award

---

You won! 🥇